Source/WebKit/UIProcess/API/APIUIClient.h

```diff
@@ -217,6 +217,11 @@ class UIClient {
     {
         completionHandler(true);
     }
+
+    virtual void requestWebAuthenticationConditonalMediationRegistration(const WTF::String&&, CompletionHandler<void(bool)>&& completionHandler)
```

Comment:
is the `WTF::` needed here? it typically isn't

Reply:
also is a `const` `&&` a thing?

Reply:
Yes the `WTF::` is needed and matches the rest of the file. Good catch with the `const` though.